packages/coinstac-simulator/src/boot-remote.js

```diff
@@ -32,6 +36,7 @@ function run({
       path.join(__dirname, 'remote.js'),
       {
         cwd: path.dirname(computationPath),
+        execArgv: process.execArgv.filter(arg => !arg.includes('--debug')),
```

Comment:
Including any sort of `--debug` flag to the master process means all forked child processes get the flag. This crashes the debugger. Perhaps the [new Node.js `--inspect` flag](https://nodejs.org/api/debugger.html#debugger_v8_inspector_integration_for_node_js) will fix this.